engine/src/SceneView.cpp

```diff
@@ -27,7 +27,54 @@ constexpr const char *kDeferredVertexShader = R"vertexShader(
 
     )vertexShader";
 
-constexpr const char *kDeferredFragmentShader = R"fragmentShader(
+constexpr const char *kDeferredDirectionalLightFragmentShader = R"fragmentShader(
+        #version 330 core
+
+        in vec2 texCoord;
+
+        uniform vec3 lightDirection;
+        uniform vec3 lightBaseColor;
+        uniform vec3 camPosition;
```

Comment:
OMFG this uniform was never set before! I knew there was something weird about how the specular highlights were moving in relation to the camera.

Reply:
Actually it was set above, nevermind.